src/components/Flyout/Flyout.tsx

```diff
@@ -209,14 +209,7 @@ export const Flyout: FC<FlyoutProps> = ({
                             overlayTriggerProps={overlayTriggerProps}
                             positionProps={positionProps}
                             fixedHeader={fixedHeader}
-                            onClick={
-                                onClick
-                                    ? () => {
-                                          onClick();
-                                          close();
-                                      }
-                                    : undefined
-                            }
+                            onClick={onClick ? onClick : undefined}
```

Comment:
will the dialogue still work/close correctly in the other components using it?

Reply:
Yes as far as I could see :)